app/api/migrations/migrations/165-default_empty_metadata/specs/fixtures.ts

```diff
@@ -0,0 +1,343 @@
+import { ObjectId } from 'mongodb';
+import { Entity, Fixture, Settings, Template } from '../types';
+
+const thesauriId = new ObjectId();
+const relTypeId = new ObjectId();
+const sourceTemplateId = new ObjectId();
+
+const settings: Settings[] = [
+  {
+    _id: new ObjectId(),
+    languages: [
+      {
+        key: 'en',
+        default: true,
+        label: 'English',
+      },
+      {
+        key: 'es',
+        label: 'Spanish',
+      },
+    ],
+  },
+];
+
+const template: Template = {
+  properties: [
+    {
+      _id: new ObjectId(),
+      label: 'Text',
+      type: 'text',
+      name: 'text',
+    },
+    {
+      content: thesauriId.toString(),
+      _id: new ObjectId(),
+      label: 'Select',
+      type: 'select',
+      name: 'select',
+    },
+    {
+      content: sourceTemplateId.toString(),
+      _id: new ObjectId(),
+      label: 'Relationship',
+      type: 'relationship',
+      relationType: relTypeId.toString(),
+      name: 'relationship',
+    },
+  ],
+  entityViewPage: '',
+  name: 'test_template',
+  _id: new ObjectId(),
+};
+
+const template2: Template = {
+  properties: [
+    {
+      _id: new ObjectId(),
+      label: 'Text',
+      type: 'text',
+      name: 'text',
+    },
+    {
+      content: thesauriId.toString(),
+      _id: new ObjectId(),
+      label: 'Select',
+      type: 'select',
+      name: 'select',
+    },
+  ],
+  entityViewPage: '',
+  name: 'test_template2',
+  _id: new ObjectId(),
+};
+
+const templates: Template[] = [template, template2];
+
+const correctEntities: Entity[] = [
+  {
+    _id: new ObjectId(),
+    title: 'correct_entity',
+    sharedId: 'correct_entity_sharedId',
+    template: template._id,
+    language: 'en',
+    metadata: {
+      text: [
+        {
+          value: 'text',
+        },
+      ],
+      select: [
+        {
+          value: 'A_id',
+          label: 'A',
+        },
+      ],
+      relationship: [
+        {
+          value: 'S_sharedId',
+          label: 'S',
+          type: 'entity',
+        },
+      ],
+    },
+  },
+  {
+    _id: new ObjectId(),
+    title: 'correct_entity_es',
+    sharedId: 'correct_entity_sharedId',
+    template: template._id,
+    language: 'es',
+    metadata: {
+      text: [
+        {
+          value: 'text_es',
+        },
+      ],
+      select: [
+        {
+          value: 'A_id',
+          label: 'A_es',
+        },
+      ],
+      relationship: [
+        {
+          value: 'S_sharedId',
+          label: 'S_es',
+          type: 'entity',
+        },
+      ],
+    },
+  },
+];
+
+const correctFixtures: Fixture = {
+  settings,
+  templates,
+  entities: correctEntities,
+};
+
+const fixtures: Fixture = {
```

Comment:
Can we add a case in which the entity has some metadata properties with correct values and some others to be fixed?